app/ui/download.js

```diff
@@ -107,7 +107,7 @@ module.exports = function(state, emit) {
   return html`
   <main class="main container">
     <section class="relative h-full w-full p-6 md:flex md:flex-row">
-      <div class="md:mr-6 md:w-1/2 overflow-y-scroll">${content}</div>
```

Comment:
scrolling here caused the download button to be below the fold and not visible when there's many files. I moved the scroll area to the details element instead.